components/org.wso2.carbon.identity.application.authenticator.samlsso/src/main/java/org/wso2/carbon/identity/application/authenticator/samlsso/manager/DefaultSAML2SSOManager.java

```diff
@@ -1228,4 +1229,20 @@ protected String getIssuer(AuthenticationContext context) {
         return properties.get(IdentityApplicationConstants.Authenticator.SAML2SSO.SP_ENTITY_ID);
     }
 
+    private EncryptedKey getEncryptedKey(EncryptedAssertion encryptedAssertion) throws Exception {
+
+        if (isNotEmpty(encryptedAssertion.getEncryptedData().getKeyInfo().getEncryptedKeys())) {
+            if (log.isDebugEnabled()) {
+                log.debug("EncryptedKey obtain from the Element.");
```

Comment:
This logline seems to be not making much sense. from what element the EncryptedKey was obtained?